src/content/advanced/gitops/apps/automatic_updates_appcr.md

```diff
@@ -0,0 +1,250 @@
+---
+linkTitle: Enable automatic updates in Apps
+title: Enable automatic updates in Apps
+description: Learn how to enable and configure automatic updates in Apps deployed using GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-automatic-update
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I enable and configure auto update for apps deployment with GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+# Enable automatic updates of an existing App
+
+Follow the below instructions to tell Flux to automatically update an existing App. Automatic update means that Flux will scan a remote repository for you and automatically update your deployed application version while also creating commits in your repository to reflect these changes.
+
+Configuring automated version updates requires a few additional Flux API objects to be defined, namely:
+
+- `ImageUpdateAutomation`, which defines the core update properties, like the git repo where the commits are to be made, commit messages structure and how to update files in the repository,
+- `ImageRepository` to configure where images are stored and new versions can be detected,
+- `ImagePolicy` to configure which of the detected new tags will be considered as possible to update to.
+- Additionally, you have to edit your existing App manifest to include a marker showing the manifest's property you want to have auto updated by Flux (usually App version property).
+
+Flux will watch for new Docker image tags for your App and use them to update the `.spec.version` field in the App CR. It will do it by pushing commits to this repository.
+
+**Note**, in order to use this mechanism you have to make sure that image tags of your App corresponds to its version, otherwise this process will result in setting meaningless version in the `.spec.version` field.
```

Comment:
```suggestion
**__Note__**, in order to use this mechanism you have to make sure that image tags of your App correspond to its version, otherwise this process will result in setting a meaningless version in the `.spec.version` field.
```